bin/playwright.sh

```diff
@@ -10,4 +10,4 @@ export PLAYWRIGHT_VERSION=$(version)
 
 echo Running Playwright v$PLAYWRIGHT_VERSION as $USER_ID with Playwright arguments $PLAYWRIGHT_ARGS
 
-docker-compose -f docker-compose.playwright.yml up --abort-on-container-exit --exit-code-from playwright
+docker-compose -f docker-compose.playwright.yml up --build --force-recreate --abort-on-container-exit --exit-code-from playwright
```

Comment:
I understand we need to rebuild the container because of the bump in playwright but won't this build the container every time the tests are run? 🚨 It is _not_ a small image.

Reply:
It will just cause the `Dockerfile.playwright` build to re-run but all the layers are still cached, so if the build results in a no-op then it's only a few milliseconds slower than not re-building.

The reason to do this is so that when the playwright version changes our local machines will actually use the new version. Docker-compose `up` doesn't rebuild the image when arguments to the Dockerfile change (like the PLAYWRIGHT_VERSION one).

If you try it out locally you'll find that the first time, yes it will be slow because we're getting a new playwright version, but subsequent runs are just as fast as they were before because the build is cached.